### PR TITLE
changed direction attribute for fillable fields

### PR DESCRIPTION
**Pull request by devgurjar**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/tabsontop.html [hunk 1]
```diff
@@ -36,7 +36,7 @@
      <div data-sly-call="${longDescription.longDescription @componentId=panel.id, longDescription=panel.description, bemBlock='cmp-tabs'}" data-sly-unwrap></div>
 
  <ol data-sly-test="${tabs.items && tabs.items.size > 0}"
-     data-sly-list.tab="${tabs.items}"
+     data-sly-list.tab="${panel.items}"
```
Comment (sakshi-arora1) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/tabsontop.html:39]: This works without updating panel sling model? If so do we still need to initialise tab sling model?

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/verticaltabs.html [hunk 2]
```diff
@@ -36,7 +36,7 @@
      <div data-sly-call="${longDescription.longDescription @componentId=panel.id, longDescription=panel.description, bemBlock='cmp-verticaltabs'}" data-sly-unwrap></div>
 <div class="cmp-verticaltabs__tabs-container" id="${widgetId}">
  <ol data-sly-test="${tabs.items && tabs.items.size > 0}"
-     data-sly-list.tab="${tabs.items}"
+     data-sly-list.tab="${panel.items}"
```
Comment (sakshi-arora1) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/verticaltabs.html:39]: same comment as given for tabs on top

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/tabsontop.html (hunk 1 above)
Comment (devgurjar) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/tabsontop.html:39]: tabs.accessibilityLabel and tabs.activeItem are coming from tabs, so we still needs tabs sling model.

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/verticaltabs.html (hunk 2 above)
Comment (devgurjar) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/verticaltabs.html:39]: tabs.accessibilityLabel and tabs.activeItem are coming from tabs, so we still needs tabs sling model.

**Review by sakshi-arora1 — APPROVED** [review]

**Pull request merged**